web-common/src/layout/RillDeveloperLayout.svelte

```diff
@@ -36,7 +39,7 @@
       commitHash: config.build_commit,
     });
 
-    return resourcesStore.init(config.instance_id);
+    return fileArtifactsStore.init(queryClient, config.instance_id);
```

Comment:
Can we move this to `web-local/src/routes/(application)/+layout.svelte`? It feels very related to the Watch Flies/Resources clients

Reply:
Further, do we need to formally initialize this store at all? Instead, will the Watch Files/Resources clients return all files/resources when they start-up, and we can populate the store with these responses as they stream in?